includes/net-standard-table.md

```diff
@@ -1,6 +1,6 @@
 | .NET Standard              | [1.0]  | [1.1]  | [1.2] | [1.3] | [1.4] | [1.5]              | [1.6]              | [2.0]               | [2.1] |
 |----------------------------|--------|--------|-------|-------|-------|--------------------|--------------------|---------------------|---------------------
-| .NET Core                  | 1.0    | 1.0    | 1.0   | 1.0   | 1.0   | 1.0                | 1.0                | 2.0                 | 3.0 |
+| .NET Core and .NET 5       | 1.0    | 1.0    | 1.0   | 1.0   | 1.0   | 1.0                | 1.0                | 2.0                 | 3.0 |
```

Comment:
I'm actually adding .NET as a new line to the .NET website. Should we be consistent?

https://github.com/dotnet/website/pull/2351

Reply:
Yes, I'll start a new PR to do that.

Reply:
On second thought that would change the structure of the table or make its labeling inaccurate --

>  The rows represent the different .NET implementations.

.NET Core and .NET 5 are the same implementation of .NET, so putting them on separate lines would be misleading.

Reply:
this would make https://dotnet.microsoft.com/platform/dotnet-standard and docs mismatch. Also, you're adding a product version into the label, when .NET is itself a target.

/cc @terrajobst

Reply:
You have the same problem with the table on https://dotnet.microsoft.com/platform/dotnet-standard -- the column header for the first column is ".NET Implementation"  -- not "target" as in TFM.  The table as a result presents ".NET Core" and ".NET" as two separate implementations of .NET when in fact they are one and the same. That's likely to cause confusion.  We should either tweak the column heading (in the dotnet site) and the table description (in the docs.microsoft.com site) or use just one line for Core and 5 in both places.

Reply:
Another thought occurred to me -- do we still need two versions of this table, or could we remove the docs.ms.com one and just link to the dotnet site one from the docs.ms.com article?

Reply:
So, what will you do when .NET 6 comes out? Add it to the name of the implementation even though we only list minimum versions?
I agree they are the same implementation, but now they have different names, so I thought it warranted a new line.
I could include this in a series of user studies that we're planning to do as well.
I'd be concerned about removing this table from the docs. I hate having to make customers go to another place to find the information they need. This article has 30k page views last month and only ~500 clicked on the interactive table link.

Reply:
I think I agree with @tdykstra. We should treat .NET 5 as rebranded .NET Core 4, that is it should be in the row as .NET Core.

Reply:
Cool. Then I'll tweak the marketing site so both sites match.